src/main/java/io/pakland/mdas/githubstats/application/FetchAvailableOrganizations.java

```diff
@@ -3,11 +3,9 @@
 import io.pakland.mdas.githubstats.application.exceptions.HttpException;
 import io.pakland.mdas.githubstats.domain.Organization;
 import io.pakland.mdas.githubstats.domain.repository.OrganizationExternalRepository;
-import org.springframework.stereotype.Service;
 
 import java.util.List;
 
-@Service
```

Comment:
This was causing an error on the constructors of the services annotated with `@Service` when the application starts